lib/structures/CommandInteraction.js

```diff
@@ -6,35 +6,36 @@ const User = require("./User");
- * @prop {String?} data.target_id The ID the of user or message targetted by a context menu command
- * @prop {Number} data.type The [command type](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-types)
- * @prop {Object?} data.resolved converted users + roles + channels
- * @prop {Collection<Channel>?} data.resolved.channels converted channels
- * @prop {Collection<Member>?} data.resolved.members converted members
- * @prop {Collection<Role>?} data.resolved.roles converted roles
- * @prop {Collection<User>?} data.resolved.users converted users
- * @prop {String?} guildID The ID of the guild in which the interaction was created
- * @prop {Member?} member The member who triggered the interaction (This is only sent when the interaction is invoked within a guild)
- * @prop {User?} user The user who triggered the interaction (This is only sent when the interaction is invoked within a dm)
- */
+* Represents an application command interaction. See Interaction for more properties.
+* @extends Interaction
+* @prop {Permission?} appPermissions The permissions the app or bot has within the channel the interaction was sent from
+* @prop {PrivateChannel | TextChannel | NewsChannel} channel The channel the interaction was created in. Can be partial with only the id if the channel is not cached.
+* @prop {Object} data The data attached to the interaction
+* @prop {String} data.id The ID of the Application Command
+* @prop {String} data.name The command name
+* @prop {Number} data.type The [command type](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-types)
+* @prop {String?} data.target_id The id the of user or message targeted by a context menu command
+* @prop {Array<Object>?} data.options The run Application Command options
+* @prop {String} data.options[].name The name of the Application Command option
+* @prop {Number} data.options[].type Command option type, 1-11
+* @prop {(String | Number | Boolean)?} data.options[].value The value of the run Application Command (Mutually exclusive with options)
+* @prop {Array<Object>?} data.options[].options The run Application Command options (Mutually exclusive with value)
+* @prop {Object?} data.resolved converted users + roles + channels
+* @prop {Collection<User>?} data.resolved.users converted users
+* @prop {Collection<Member>?} data.resolved.members converted members
+* @prop {Collection<Role>?} data.resolved.roles converted roles
+* @prop {Collection<Channel>?} data.resolved.channels converted channels
+* @prop {String?} guildID The ID of the guild in which the interaction was created
+* @prop {Member?} member The member who triggered the interaction (This is only sent when the interaction is invoked within a guild)
+* @prop {User?} user The user who triggered the interaction (This is only sent when the interaction is invoked within a dm)
+*/
```

Comment:
Was this sorted in a recent PR and I've unsorted it?

Reply:
Yes